sdgx/data_processors/transformers/fixed_combination.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import pandas as pd
+
+from sdgx.data_models.metadata import Metadata
+from sdgx.data_processors.extension import hookimpl
+from sdgx.data_processors.transformers.base import Transformer
+from sdgx.utils import logger
+
+
+class FixedCombinationTransformer(Transformer):
+    """
+    A transformer that handles columns with fixed combinations in a DataFrame.
+
+    This transformer identifies and processes columns that have fixed relationships (high covariance) in a given DataFrame.
+    It can remove these columns during the conversion process and restore them during the reverse conversion process.
+
+    Attributes:
+        fixed_combinations (dict[str, set[str]]): A dictionary mapping column names to sets of column names that have fixed relationships with them.
+    """
+
+    fixed_combinations: dict[str, set[str]] = {}
```

Comment:
@MooooCat This may cause `self.fixed_combinations` using class member and shared with all instance

see: https://github.com/hitsz-ids/synthetic-data-generator/pull/232#issuecomment-2461477585 and we have fixed some of them in this pr